frontend/src/pages/Resources/components/ResourceSharingForm/ResourceSharingForm.jsx

```diff
@@ -90,355 +139,401 @@ export function ResourceSharingForm(props) {
+                  >
+                    {formerrors["url"] ? (
+                      <div>* {formerrors["url"]}</div>
+                    ) : (
+                      <div>&nbsp; &nbsp;</div>
+                    )}
+                  </div>
                 </div>
               </div>
-            </div>
 
-            <div className={`form-group ${style["form-group"]}`}>
               <div
                 className={
                   dark
                     ? `${style["resource-input"]} ${style["resource-input-dark"]} `
                     : `${style["resource-input"]} ${style["resource-input-light"]}`
                 }
               >
-                <input
-                  placeholder="Resource URL"
-                  id="txt_link"
-                  type="url"
-                  name="link"
+                <textarea
+                  placeholder="Resource Description"
+                  id={style["txt-desc"]}
+                  rows="6"
+                  cols="20"
+                  name="description"
                   onChange={handleChange}
-                />
-                <i className="fas fa-link"></i>
+                ></textarea>
+                <i className="fas fa-comment-dots"></i>
                 <div
                   className={`${style["validation"]} validation d-sm-none d-md-block`}
                 >
-                  {formerrors["link"] ? (
-                    <div>* {formerrors["link"]}</div>
+                  {formerrors["description"] ? (
+                    <div>* {formerrors["description"]}</div>
                   ) : (
                     <div>&nbsp; &nbsp;</div>
                   )}
                 </div>
               </div>
-            </div>
-
-            <div
-              className={
-                dark
-                  ? `${style["resource-input"]} ${style["resource-input-dark"]} `
-                  : `${style["resource-input"]} ${style["resource-input-light"]}`
-              }
-            >
-              <textarea
-                placeholder="Resource Description"
-                id={style["txt-desc"]}
-                rows="6"
-                cols="20"
-                name="description"
-                onChange={handleChange}
-              ></textarea>
-              <i className="fas fa-comment-dots"></i>
               <div
-                className={`${style["validation"]} validation d-sm-none d-md-block`}
+                className={
+                  dark
+                    ? `${style["resource-input1"]} ${style["resource-input1-dark"]} `
+                    : `${style["resource-input1"]} ${style["resource-input1-light"]}`
+                }
               >
-                {formerrors["description"] ? (
-                  <div>* {formerrors["description"]}</div>
-                ) : (
-                  <div>&nbsp; &nbsp;</div>
-                )}
-              </div>
-            </div>
-            <div
-              className={
```

Comment:
Any reason of removing these?

Reply:
Hello @Kajol-Kumari, I haven't removed any existing code from the html part. I have only changed some key values from the 'formdata part', 'schema part' and 'keys from the html error code part as per the values in formdata and schema'. 
Please recheck.